documentation/modules/configuring/con-scaling-kafka-clusters.adoc

```diff
@@ -40,11 +40,15 @@ spec:
  # ...
 ----
 
+When adding brokers through the `Kafka` configuration, node IDs start at 0 (zero) and the Cluster Operator assigns the next lowest ID to a new node. 
+The broker removal process starts from the broker pod with the highest ID in the cluster.
+
+If you are managing nodes in the cluster using the the preview of the node pools feature, you adjust the `KafkaNodePool.spec.replicas` configuration to change the number of nodes in the node pool.
+With node pools, you can  xref:proc-managing-node-pools-ids-{context}[assign node IDs for scaling operations].  
```

Comment:
Should we make it more clear here that even creating a new cluster is in a way _scaling operation_ (from nothing to N nodes)? I think some people might confuse this test right now as that they can use it only when scaling existing clusters.

Reply:
Changed as follows:

> If you are managing nodes in the cluster using the the preview of the node pools feature, you adjust the `KafkaNodePool.spec.replicas` configuration to change the number of nodes in the node pool.
> Additionally, when scaling existing clusters with node pools, you can xref:proc-managing-node-pools-ids-{context}[assign node IDs for the scaling operations].